lib/compress/fse_compress.c

```diff
@@ -337,6 +337,67 @@ size_t FSE_writeNCount (void* buffer, size_t bufferSize,
     return FSE_writeNCount_generic(buffer, bufferSize, normalizedCounter, maxSymbolValue, tableLog, 1 /* write in buffer is safe */);
 }
 
+size_t FSE_estimateNCountSize(const short* normalizedCounter, unsigned maxSymbolValue, unsigned tableLog)
+{
```

Comment:
This code seems repeated from `FSE_writeNCount_generic()`.
Does it have to be ?